### ISSUE TITLE
Poor detection of words with pretrained models

### ISSUE THREAD

**Issue by rmdmohan20** [open-mmlab/mmocr#636]
Hi,
I have been trying to work on **detecting words in images of invoices**.
Word detection is working for images with smaller dimensions. But it is not detecting higher dimension images. 
I read in [354](https://github.com/open-mmlab/mmocr/issues/354) and [274](https://github.com/open-mmlab/mmocr/issues/274)  that if the images is cropped into smaller pieces it is working fine. But i **want to detect word on an whole image**.
If training needs to be done with more images, **what would be the ideal number to chose and in what basis should the training be, to get a good results with detection.**

**Comment by cuhk-hbsun** [comment]
1. What is the image size you use in the config file? Is it something like the input image size?
2. I suggest that try to finetune the model on your own data. The more images treated as trainset, the better result you will get.

**Comment by rmdmohan20** [comment]
Thanks for the reply.
1. We used default image size in the config.
2. I observed the output for images in different ranges of dimensions and i found out that images with resolution around 500pixels, detections are good. as the resolution increases detection goes wrong.
Attaching the images below, might be useful
**907x791 pixels**
![526_70%_fcenet_pretrain](https://user-images.githubusercontent.com/59682565/144825641-4629687c-b218-4b76-ba45-05d560fbf1fa.jpg)
**389x339 pixels**
![526_30%_fcenet_pretrain](https://user-images.githubusercontent.com/59682565/144825647-ffc50992-0e45-4c2f-8312-30a65728c716.jpg)
Here we can see the detections are better for second image. To improve detections for all dimension, will it be a good start with 10000 images?
